modules.d/62bluetooth/module-setup.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# called by dracut
+check() {
+  require_binaries /usr/libexec/bluetooth/bluetoothd || return 1
+
+  return 255
+}
+
+depends() {
+  echo systemd dbus
+  return 0
+}
+
+installkernel() {
+  instmods bluetooth btrtl btintel btbcm bnep ath3k btusb rfcomm
+  inst_multiple -o \
+    /usr/lib/firmware/ar3k/AthrBT* \
+    /usr/lib/firmware/ar3k/ramps* \
+    /usr/lib/firmware/ath3k-1.fw \
+    /usr/lib/firmware/BCM2033-MD.hex \
+    /usr/lib/firmware/bfubase.frm \
+    /usr/lib/firmware/BT3CPCC.bin \
+    /usr/lib/firmware/brcm/*.hcd \
+    /usr/lib/firmware/mediatek/mt7622pr2h.bin \
+    /usr/lib/firmware/qca/nvm* \
+    /usr/lib/firmware/qca/crnv* \
+    /usr/lib/firmware/qca/rampatch* \
+    /usr/lib/firmware/qca/crbtfw* \
+    /usr/lib/firmware/rtl_bt/* \
+    /usr/lib/firmware/intel/ibt* \
+    /usr/lib/firmware/ti-connectivity/TIInit_* \
+    /usr/lib/firmware/nokia/bcmfw.bin \
+    /usr/lib/firmware/nokia/ti1273.bin
+}
+
+install() {
+  inst_multiple \
+    /usr/libexec/bluetooth/bluetoothd \
+    /usr/lib/systemd/system/bluetooth.target \
+    /usr/lib/systemd/system/bluetooth.service \
+    /etc/bluetooth/main.conf \
```

Comment:
What about other config files:
```console
$ strings /usr/libexec/bluetooth/bluetoothd|fgrep etc/bluetooth
/etc/bluetooth/network.conf
/etc/bluetooth/input.conf
/etc/bluetooth/main.conf
```

Reply:
@haraldh Those files are not required for bluetooth to work. What's needed is 3 things:
 i) main.conf from bluez, 
ii) relevant files (like firmware, software, modules) to enable BT generally, which includes bluez and possibly d-bus, 
iii) the device profile so bluez actually connects to something during boot
All 3 are already present in this PR.